docusaurus/docs/how-to-guides/data-source-plugins/sql-requirements.md

```diff
@@ -0,0 +1,125 @@
+---
+id: sql-requirements
+title: Requirements to support SQL expressions
+sidebar_label: Support SQL expressions
+description: Requirements for data source plugins to support SQL expressions.
+keywords:
+  - grafana
+  - plugin
+  - datasource
+  - sql
+---
+
+Follow this guide to understand what your data source plugin requires to support SQL expressions.
+
+:::caution
+Only plugins with a backend can support SQL expressions.
+:::
+
+Requirements depend on the type of response for your query. SQL expressions support two types of responses:
+
+- [Tabular data response](#requirements-for-tabular-data-responses) 
+- [Labeled metric time data response](#requirements-for-labeled-metric-time-data-responses) 
+
+## Requirements for tabular data responses
+
+A tabular data response is a single [data frame](../../key-concepts/data-frames) with no labels on any of the fields (columns). In short, a data frame that can be directly mapped to a SQL table.
+
+Tabular data responses work out-of-the-box.
+
+## Requirements for labeled metric data responses
+
+A labeled metric time data response contains data that meets the [data plane specification](https://grafana.com/developers/dataplane) and has the `Frame.Meta.Type` property set with the data plane `type`. It can be a timeseries or numeric. 
```

Comment:
```suggestion
A labeled metric time data response contains data that meets the [data plane specification](https://grafana.com/developers/dataplane) and has the `Frame.Meta.Type` property set with the data plane type. It can be a timeseries or numeric. 
```